plasmapy/simulation/particle_tracker.py

```diff
@@ -555,6 +645,12 @@ def _stop_particles(self, particles_to_stop_mask):
         self.v[particles_to_stop_mask] = np.NaN
 
     def _remove_particles(self, particles_to_remove_mask):
+        """Remove the specified particles from the simulation.
+
+        For the same of keeping consistent array lengths, the position and velocities of the
+        particles are set to NaN.
```

Comment:
"of the removed particles" 

Does it say anywhere explicitly that 

- x,v != NaN -> Tracked particle
- x != NaN, v=NaN -> Stopped particle (still in the simulation, but position is no longer evolved). 
- x,v = NaN -> Removed particle